functions/go/set-namespace/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.13-stretch
```

Comment:
go.mod uses go 1.14 so can this use the same?

Reply:
This file is generated with `go run ./main.go gen ./`: https://googlecontainertools.github.io/kpt/guides/producer/functions/golang/#publish-the-function

Reply:
Done